js/box-colors.js

```diff
@@ -0,0 +1,25 @@
+window.onload = () => {
```

Comment:
Really cool stuff man! You're way ahead. The code looks great, but it's not running in the git preview. I'll have to check it out by cloning it onto my local machine.